app/boards/shields/swept/Kconfig.shield

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Pete Johanson
```

Comment:
Need to update the copyright headers to `2023 The ZMK Contributors` across the files.

Reply:
Ok. I'll fix that.